lib/clients/jira.go

```diff
@@ -349,7 +352,12 @@ func (j realJIRAClient) request(f func() (interface{}, *jira.Response, error)) (
 	b := backoff.NewExponentialBackOff()
 	b.MaxElapsedTime = j.config.GetTimeout()
 
-	er := backoff.Retry(op, b)
+	er := backoff.RetryNotify(op, b, func(err error, duration time.Duration) {
```

Comment:
Yeah, I should probably find better names. In the outer scope `err` is the error returned from the API function, `er` is the error returned from `RetryNotify()`. In the inner, anonymous scope, `err` shadows the outer scope variable, and is the error from the API function provided by the function arguments.